devise-jwt.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'devise', '~> 4.0'
+  spec.add_dependency 'devise', '>= 4.0'
```

Comment:
I know i've made this change in this PR a while ago : ( https://github.com/waiting-for-dev/devise-jwt/pull/275 )

But I think we're finally ready to support devise 5. and seeing that they can cut a release at any moment ( based on the changelog i've sent below) @waiting-for-dev